README.md

```diff
@@ -47,11 +47,50 @@ Fb::Page#posts
 Given a page with posts, you can get the posts on the page since creation by calling:
 
 ```ruby
-page = Fb::Page.new access_token: '--valid-access-token--'
+user = Fb::User.new access_token: '--valid-access-token--'
+page = user.pages.first
 page.posts
 # => [#<Fb::Post: id="1234", type="video">, #<Fb::Post: id="5678", type="video">]
 ```
 
+Fb::Page#like_count
+--------------
+
+Given a page with insights data (verify on the insights section of the page), you can get the the number of likes for the page.
+
+```ruby
+user = Fb::User.new access_token: '--valid-access-token--'
+page = user.pages.first
+page.like_count
+# => 15000
+```
+
+Pass an `until` option to aggregate the data only up to a certain date.
```

Comment:
@S1v4 `aggregate` is not the right word here since you are just "getting" a like count.

The aggregation is just an implementation detail that we are hiding.